File: FrontEnd/MshopUI/src/app/models/supplier.ts
import { BaseModel } from './base-model/base-model';

export class Supplier extends BaseModel{
    SupplierID: string;
    SupplierCode: string;
    SupplierName: string;
    SupplierPhone: string;
    SupplierAddress: string;
    BankAccount: string;
    BankName: string;
    BankBranch: string;
    ContactName: string;
    ContactPhone: string;
    ContactEmail: string;
    ContactAddress: string;
}

File: BackEnd/Models/Dictionary/Supplier.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Models
{
    public class Supplier:BaseModel
    {
        public Guid SupplierID { get; set; }
        public string SupplierCode { get; set; }
        public string SupplierName { get; set; }
        public string SupplierAddress { get; set; }
        public string BankAccount { get; set; }
        public string BankName { get; set; }
        public string BankBranch { get; set; }
        public string SupplierPhone { get; set; }
        public string ContactName { get; set; }
        public string ContactPhone { get; set; }
        public string ContactEmail { get; set; }
        public string ContactAddress { get; set; }

    }
}
